### ISSUE TITLE
NTR: Apolipoprotein B high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8459]
New term request Apolipoprotein B high in CSF (elthat_101120221551): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000204
- label: Elevated CSF apolipoprotein B concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: PMID:34590423
- def: The concentration of apolipoprotein B in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8459
- synonyms: Apolipoprotein B high in CSF(NONE)
